src/main/java/com/xceptance/xlt/report/ReportGenerator.java

```diff
@@ -162,9 +162,6 @@ public ReportGenerator(final FileObject inputDir, final File outputDir, final bo
         // configure the thread pool
         TaskManager.getInstance().setMaximumThreadCount(config.getThreadCount());
 
-        // configure the PNG encoder
-        JFreeChartUtils.setPngCompressionLevel(config.getChartCompressionLevel());
```

Comment:
I would like to keep the opportunity to set the desired compression level/quality, in modified form of course, i.e. as data type double with 0.75 as the default. Please rework `ReportGeneratorConfiguration` and `config/reportgenerator.properties` accordingly.

Reply:
Done